pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -202,6 +202,9 @@ type VolumeBindingArgs struct {
 	// Value must be non-negative integer. The value zero indicates no waiting.
 	// If this value is nil, the default value will be used.
 	BindTimeoutSeconds int64
+
+	// Shape specifies the shape points defining score function shape.
+	Shape []UtilizationShapePoint
```

Comment:
I'm not familiar with the scheduler APIs. Does it follow standard Kubernetes API conventions where alpha fields need to be feature gated?

Reply:
Otherwise this lgtm

Reply:
this is similar to any other object api, we can add fields, but removing them requires a new upping the version.